src/providers/postgres/qgspostgresconn.cpp

```diff
@@ -930,6 +930,35 @@ QString QgsPostgresConn::quotedIdentifier( QString ident )
   return ident;
 }
 
+static QString quotedString( QString v )
```

Comment:
In cases like this I normally change the argument to a const reference and then make the first line copy the string to a temporary variable. The copy is cheap, and having the const ref in the signature just "looks right" to me. Maybe that's just from looking too long at the qt api though...!